src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -2437,6 +2431,96 @@ def update_policy_setdefinition(cmd, policy_set_definition_name, definitions=Non
     return policy_client.policy_set_definitions.create_or_update(policy_set_definition_name, parameters)
 
 
+def create_policy_exemption(cmd, name, policy_assignment=None, exemption_category=None,
+                            policy_definition_reference_ids=None, expires_on=None,
+                            display_name=None, description=None, resource_group_name=None, scope=None,
+                            metadata=None):
+    if policy_assignment is None:
+        raise RequiredArgumentMissingError('--policy_assignment is required')
+    if exemption_category is None:
+        raise RequiredArgumentMissingError('--exemption_category is required')
+
+    policy_client = _resource_policy_client_factory(cmd.cli_ctx)
+    scope = _build_policy_scope(policy_client.config.subscription_id,
+                                resource_group_name, scope)
+    PolicyExemption = cmd.get_models('PolicyExemption')
+    exemption = PolicyExemption(policy_assignment_id=policy_assignment, policy_definition_reference_ids=policy_definition_reference_ids,
+                                exemption_category=exemption_category, expires_on=expires_on,
+                                display_name=display_name, description=description, metadata=metadata)
+    createdExemption = policy_client.policy_exemptions.create_or_update(scope, name, exemption)
+    return createdExemption
+
+
+def update_policy_exemption(cmd, name, exemption_category=None,
+                            policy_definition_reference_ids=None, expires_on=None,
+                            display_name=None, description=None, resource_group_name=None, scope=None,
+                            metadata=None):
+    policy_client = _resource_policy_client_factory(cmd.cli_ctx)
```

Comment:
the same

Reply:
cf_policy_exemptions is not defined in this file. And I needs subscription_id from the policy_client object. It doesn't help remove this line.